package.json

```diff
@@ -80,7 +80,7 @@
     "expect.js": "^0.3.1",
     "fs-extra": "^7.0.1",
     "glob": "^7.1.3",
-    "knex": "0.x",
+    "knex": "0.16.4-next2",
```

Comment:
Disregard the "next" status of it, it's definitely the best available Knex version at this point.

Reply:
too bad that the 0.16.4 isn't out yet